### PR TITLE
coverage ui

### PR DESCRIPTION
**Pull request by mullr**
- Add hooks for spec coverage display
- Consistently use Headline Case for Menu Items
- API rev and some manual HTML templating
- Flesh out spec UI a bit
- Remove spurious '.rs' file
- Update folder metadata on folder/file changes
- Show/hide spec versions, results
- Add batch-eval support to conform spec command
- Further polish on specs panel
- Fix batch eval
- Add some more multi-select commands
- Disable unimplemented menu commands
- Flesh out spec coverage UI
- Use api to update spec icon

### CODE REVIEW

File: vscode/src/config.ts [hunk 1]
```diff
@@ -145,7 +145,10 @@ export function toolPath(tool_name: string): string {
         toolPath = firstExistingPath(
             customPath,
             path.join("/usr/local/bin/", tool_name),
-            path.join("/usr/bin/", tool_name)
+            path.join("/usr/bin/", tool_name),
+            // TODO - DEBUG - DELETE - once more general
+            path.join("/home/zack/.cargo/bin/", tool_name)
```
Comment (jonlamb-gh) [inline vscode/src/config.ts:150]: :axe:

File: vscode/src/specs.ts [hunk 2]
```diff
@@ -1,121 +1,586 @@
...
         public specVersion: api.SpecVersionId,
-        public specEvalResultId: api.SpecEvalResultId
-    ) {}
+        workspaceState: SpecsTreeMemento
+    ) {
+        super("Spec Version: " + specVersion);
+        if (workspaceState.getShowResults()) {
+            super.iconPath = new vscode.ThemeIcon("file", new vscode.ThemeColor("symbolIcon.constructorForeground"));
+        } else {
+            // TODO color this based on if it has been executed, and if it was successful or not
```
Comment (jonlamb-gh) [inline vscode/src/specs.ts:416]: I added some top-level-stats-icon handling for this on the `spec-status` branch. I can follow up with that once this lands if it's easier.
https://github.com/auxoncorp/speqtr-support/tree/spec-status

**Review by jonlamb-gh — CHANGES_REQUESTED** [review]
The local config path changes need reverted, otherwise LGTM

File: vscode/src/specs.ts (hunk 2 above)
Comment (mullr) [reply vscode/src/specs.ts:420]: oh man I missed that branch somehow, I'm sorry. I'll pull it in here.

**Review by jonlamb-gh — APPROVED** [review]

**Pull request merged**